packages/core/src/app/customer/CheckoutButtonContainer.tsx

```diff
@@ -27,22 +37,63 @@ const CheckoutButtonContainer: FunctionComponent<CheckoutButtonsListOnTopProps &
         },
     } = checkoutState;
     const config = getConfig();
+    const methodIds = config?.checkoutSettings.remoteCheckoutProviders ?? [];
+
+    useEffect(() => {
+        const changeState = setTimeout(()=>{
+            if(!isInitializingCustomer()) {
+                setIsLoading(false);
+            }
+        },100);
+
+        return () => {
+            clearTimeout(changeState);
+            setIsLoading(true);
+        }
+    }, [isInitializingCustomer]);
+
+    if (!config || methodIds.length === 0) {
+        return null;
+    }
+
+    const supportedMethodIds = sortMethodIds(filterUnsupportedMethodIds(methodIds));
+    const buttonsCount = supportedMethodIds.length;
+
+    if (buttonsCount < 1) {
+        return null;
+    }
 
-    if (!config) {
+    try {
+        checkEmbeddedSupport(supportedMethodIds);
+    } catch (error) {
         return null;
     }
 
     return (
-        <div className='wallet-buttons-container'>
-            <CheckoutButtonListV1
-                checkEmbeddedSupport={checkEmbeddedSupport}
-                copywritingStringId='remote.start_with_text'
-                deinitialize={checkoutService.deinitializeCustomer}
-                initialize={checkoutService.initializeCustomer}
-                isInitializing={isInitializingCustomer()}
-                methodIds={config.checkoutSettings.remoteCheckoutProviders}
-                onError={onUnhandledError}
-            />
+        <div className='checkout-buttons-container'>
+            <p>
+                <TranslatedString id="remote.start_with_text" />
+            </p>
+            <div className={classNames({
+                'checkout-buttons--1': buttonsCount === 1,
+                'checkout-buttons--2': buttonsCount === 2,
+                'checkout-buttons--3': buttonsCount === 3,
+                'checkout-buttons--4': buttonsCount === 4,
+                'checkout-buttons--5': buttonsCount === 5,
+                'checkout-buttons--n': buttonsCount > 5,
+            })}>
+                <WalletButtonsContainerSkeleton buttonsCount={buttonsCount} isLoading={isLoading}>
+                    <CheckoutButtonListV1
```

Comment:
When are we switching over to `CheckoutButtonListV2`? We want the buttons to load from payment integration packages. cc @animesh1987

Reply:
To use `CheckoutButtonListV2`, we must complete the locale work which we discussed in a previous meeting. That's why we are still using the old ` CheckoutButtonList` now.

Reply:
Ah thanks Peng, I missed the previous meeting 😅

Reply:
Yes, we had a ticket to switch the buttons at top to V2. Then realised we need to complete the locale work in order to enable it. We have moved it as part of the react project for now and should be able to handle it then.